src/main/java/org/folio/fqm/resource/FqlQueryController.java

```diff
@@ -44,6 +45,13 @@ public ResponseEntity<List<Map<String, Object>>> getContents(ContentsRequest con
       contentsRequest.getFields(), contentsRequest.getIds()));
   }
 
+  /**
+   * This endpoint is deprecated and will be removed in a future version.
+   * @deprecated Since version 3.0.0.
+   * This endpoint is planned for removal.
+   */
+  @Deprecated(since = "3.0.0", forRemoval = true)
```

Comment:
Can we mark this as deprecated in the OpenAPI spec, too (in folio-query-tool-metadata)?

Reply:
@mweaver-ebsco  https://github.com/folio-org/folio-query-tool-metadata/pull/71